src/main/webapp/i18n/de/programmingExercise.json

```
@@ -142,7 +142,9 @@
                     "updated": "Die Testfälle wurden erfolgreich gespeichert.",
                     "couldNotBeUpdated": "Die folgenden Testfälle konnten nicht gespeichert werden: {{testCases}}.",
                     "weightSumError": "Die Summe aller Testfall-Gewichte beträgt 0 oder weniger. Bitte stelle mindestens ein Testfall-Gewicht größer als 0 ein.",
-                    "showInactive": "Inaktive Testfälle anzeigen"
+                    "showInactive": "Inaktive Testfälle anzeigen",
+                    "settingNull": "Der Testfall {{testCase}} darf keine Einstellungen haben, die null sind.",
```

```suggestion
                    "settingNull": "Der Testfall {{testCase}} darf keine leeren Einstellungen haben.",
```
I think having `null` in the user-facing error message is problematic, as it results in an easy confusion between the meanings as nullpointer and zero, the latter being allowed.